examples/fluent-bit/ecs-log-deletion/Dockerfile

```diff
@@ -0,0 +1,14 @@
+# Use the standard Amazon Linux base, provided by ECR/KaOS
+# This is a placeholder, replaced by the `DockerFileBuild` build system.
+# It will point to the standard shared Amazon Linux image, with a versioned tag.
+FROM AMAZON_LINUX
+
+# Install Cron
+RUN yum install -y cronie which findutils perl-core compat-libcap1 lsof tar shadow-utils setuptool procps && yum clean all
+
+# Copy crontab that deletes logs
+COPY cron/delete-log-files /etc/cron.d/delete-log-files
```

Comment:
if the dockerfile copies a file, it should be in the project directory so that customers can run the example simply by running `docker build`.

Reply:
got it